webapp/javascript/components/enchaners.tsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import { RootState } from '@pyroscope/redux/store';
+import { selectUIState } from '../redux/reducers/views';
+
+import { setUIValue } from '../redux/actions';
+
+export interface ICollapsible {
```

Comment:
```suggestion
export interface Collapsible {
```

Reply:
Yeah, I was bond to i-prefixed interfaces. Don't need that for sure

Reply:
Fixed